src/axom/primal/tests/primal_hexahedron.cpp

```diff
@@ -78,11 +78,30 @@ class HexahedronTest : public ::testing::Test
     qData2[5] = QPoint {1.5, 1, 3.5};
     qData2[6] = QPoint {3.5, 1, 3.5};
     qData2[7] = QPoint {3.5, 1, 1.5};
+
+    // Reproducer Test Case
+    qData3[0] =
+      QPoint {-70.83333333333334, -165.0000000000000, -238.0000000000000};
+    qData3[1] =
+      QPoint {-70.83333333333334, -143.0000000000000, -238.0000000000000};
+    qData3[2] =
+      QPoint {-52.0000000000000, -143.0000000000000, -238.0000000000000};
+    qData3[3] =
+      QPoint {-52.0000000000000, -165.0000000000000, -238.0000000000000};
+    qData3[4] =
+      QPoint {-70.83333333333334, -165.0000000000000, -221.0000000000000};
+    qData3[5] =
+      QPoint {-70.83333333333334, -143.0000000000000, -221.0000000000000};
+    qData3[6] =
+      QPoint {-52.0000000000000, -143.0000000000000, -221.0000000000000};
+    qData3[7] =
+      QPoint {-52.0000000000000, -165.0000000000000, -221.0000000000000};
```

Comment:
@adayton1 I added this test case based on the reproducer info you provided.
Unfortunately, I was not able to get this case to fail running with intel oneAPI compiler with the different release/debug/etc. build types.

Reply:
Thanks for adding this @bmhan12 

I wonder if the failure is somehow right on the boundary of $\frac{5}{6}$, and this representation is `*.83333333333334` does not capture the exact same number that @adayton1 was seeing (?)

Reply:
> I wonder if the failure is somehow right on the boundary ...

I tried changing to a partly fractional representation [here](https://github.com/LLNL/axom/pull/1186/commits/dd8cb545ec18c96fce3ac1d950c4cafca3c137a3). 
No error, but did need to generalize the significant figures check.